src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let sum = days * 40;
```

Comment:
Consider using a constant for the rental cost per day (40) for better readability and maintainability.